test/e2e/framework/.import-restrictions

```diff
@@ -0,0 +1,168 @@
+{
+	"Rules": [
+		{
+			"SelectorRegexp": "k8s[.]io/kubernetes/pkg/",

```

Comment:
importing `k8s.io/kubernetes/pkg` is the biggest problem of the framework, and we need to stop doing these if we want it to move out.
unfortunately, creating shared repos for some of these packages can end up slow and political.
(judging from the similar work of moving kubeadm out).

Reply:
Could you elaborate on this point further, please?
I would imagine that many framework utility packages would need to import something from `k8s/kubernetes/pkg` in order to run e2e tests 🤔

Reply:
Actually, never mind.. this makes a complete lot of sense: the e2e framework should depend on client code and public apis.
so changing question, I dont know if the e2e framework is at this point (there are some place where we do import some k/pkgs. Do we still remove these for now?

Reply:
we still have keep the problematic `k8s.io/kubernetes/pkg` imports in import boss, and remove them one by one.

Reply:
Yeah this PR should be scoped to listing out the current deps so this looks fine to me (yes, it's a long list)